sdcm/cluster.py

```diff
@@ -488,8 +488,12 @@ def host_id(self):
         raise AssertionError(f"Could not find the requested node {self.ip_address} in nodetool status")
 
     def refresh_ip_address(self):
+        if self.ssh_login_info["hostname"] == self.external_address:
+            return
+
         self.ssh_login_info["hostname"] = self.external_address
-        self.remoter.hostname = self.external_address
+        self.remoter.stop()
+        self._init_remoter(self.ssh_login_info)
         self._init_port_mapping()
```

Comment:
Is it intentional that you don't init port mapping if IP address is not changed?

Reply:
Yes, by @dkropachev  it's correct way

Reply:
It is already up at this point.